traitsui/testing/tester/wx/_traitsui/range_editor.py

```diff
@@ -15,12 +15,70 @@
     SimpleSliderEditor,
 )
 
-from traitsui.testing.tester import locator
-from traitsui.testing.tester.wx import _registry_helper
-from traitsui.testing.tester.wx.common_ui_targets import (
-    LocatedSlider,
-    LocatedTextbox
-)
+from traitsui.testing.tester import command, locator
+from traitsui.testing.tester.wx import _interaction_helpers, _registry_helper
+
+
+class LocatedTextbox:
+    """ Wrapper class for a located Textbox in Wx.
+
+    Parameters
+    ----------
+    textbox : Instance of wx.TextCtrl
+    """
+
+    def __init__(self, textbox):
+        self.textbox = textbox
+
+    @classmethod
+    def register(cls, registry):
+        """ Class method to register interactions on a LocatedTextbox for the
+        given registry.
+
+        If there are any conflicts, an error will occur.
+
+        Parameters
+        ----------
+        registry : TargetRegistry
+            The registry being registered to.
+        """
+        registry_helper.register_editable_textbox_handlers(
```

Comment:
```suggestion
        _registry_helper.register_editable_textbox_handlers(
```

I think the tests are catching these errors.